apps/client/src/widgets/dialogs/rosetta_warning.ts

```diff
@@ -0,0 +1,88 @@
+import BasicWidget from "../basic_widget.js";
+import { Modal } from "bootstrap";
+import utils from "../../services/utils.js";
+import { t } from "../../services/i18n.js";
+
+const TPL = /*html*/`
+<div class="rosetta-warning-dialog modal mx-auto" tabindex="-1" role="dialog" style="z-index: 2000;">
```

Comment:
![image](https://github.com/user-attachments/assets/a253c455-ac36-4368-8ded-8e9672a9cd78)

Under dark mode, the contrast for the title is not great. I suggest we remove the red background.